### PR TITLE
New Mod Manager (Update transition)

### PR DESCRIPTION
**Pull request by Sora-yx**
The new Mod Manager is ready, since it's a whole new program with a different update system, the vanilla Manager need to be updated one last time to handle the transition (and so the users can do the update without manual install.)
[A snapshot of the current loader](https://github.com/X-Hax/sadx-mod-loader/releases/tag/Snapshot-2023.30) has been made as well as a [legacy branch](https://github.com/X-Hax/sadx-mod-loader/tree/legacy) in case we need to go back and also for archive purpose.

tl;dr this PR will make the current Manager able to download and install the new Manager.

### CODE REVIEW

**Review by MainMemory — CHANGES_REQUESTED** [review]
Since this version is meant as a transitional step between the new and old mod managers, it should ideally produce a minimal download that only serves to install the new manager. Additionally, an option to download a legacy manager that won't be updated would be nice to have.
